ARE/services/WebService/src/main/java/eu/asterics/mw/webservice/SseResource.java

```diff
@@ -114,6 +115,7 @@ public static void initializeDataChannelListeners() {
     @Path("/runtime/deployment/listener")
     @GET
     @Produces(SseFeature.SERVER_SENT_EVENTS)
+    @Description("Subscribes for deployment events. The events are delivered using SSE.")
```

Comment:
I wouldn't write "The events are delivered using SSE." to every entry. Users of `SseResource.java` should aware of that fact anyway.

Reply:
I assume the description is used to generate the `/restfunctions` response. Therefore it makes sense to mention SSE in the description since the path `/runtime/deployment/listener` does not imply the usage of SSE.